doc/classes/BaseMaterial3D.xml

```diff
@@ -425,6 +426,7 @@
 			Represents the size of the [enum TextureParam] enum.
 		</constant>
 		<constant name="TEXTURE_FILTER_NEAREST" value="0" enum="TextureFilter">
+			The simplest and fastest method of filtering, but the texture will look pixelized.
```

Comment:
CC @clayjohn (also to help write the other filter constants).

Reply:
Texture filter reads from the nearest pixel only. The simplest and fastest method of filtering...

Reply:
Usually this is called "Nearest Neighbor", so maybe the description should contain those words.